src/eleveldb.erl

```diff
@@ -297,7 +297,7 @@ option_types(open) ->
      {block_size_steps, integer},
      {paranoid_checks, bool},
      {verify_compactions, bool},
-     {compression, bool},
+     {compression, any},  %% how to do a list of atoms?
```

Comment:
Looks like there is a `validate_type` function in this module, which we could augment to take a list/enumeration of valid atoms and then check to make sure that the value is in the enumeration.  Something like:

```
validate_type({_Key, [snappy, lz4]}, snappy) -> true;
validate_type({_Key, [snappy, lz4]}, lz4) -> true;
```